src/main/java/calculator/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+
+        Calculator calculator = new Calculator();
+        calculator.calculator();
```

Comment:
```suggestion
        calculator.calculate();
```
처럼 메서드 명은 동사형으로 바꿔서 무슨 동작을 하는지 정확히 하면 좋을 것 같아요

Reply:
합 감사합니다! 2주차 미션때 적용하도록 하겠습니다:)